cuda_pathfinder/pyproject.toml

```diff
@@ -0,0 +1,58 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+[project]
+name = "cuda-pathfinder"
+description = "Pathfinder for CUDA components (mainly libs, headers)"
+authors = [{ name = "NVIDIA Corporation", email = "cuda-python-conduct@nvidia.com" }]
+license = "LicenseRef-NVIDIA-SOFTWARE-LICENSE"
```

Comment:
This license would need to be updated as well

Reply:
Done: 787236c14e07e4173eced30d4066d821d6cbd016

Thanks for catching this!